app/controllers/VehicleRegistrationCountryController.scala

```diff
@@ -62,33 +57,11 @@ class VehicleRegistrationCountryController @Inject() (
               case None        => form
               case Some(value) => form.fill(value)
             }
-            renderPage(arrivalId, request.userAnswers.mrn, mode, preparedForm, countries, Results.Ok)
+            Future.successful(Ok(view(preparedForm, countries, request.userAnswers.mrn, arrivalId, mode)))
```

Comment:
Line 53 can be changed to a `map` instead of `flatMap` then the `Future.successful` can be removed here